core/ledger/kvledger/bookkeeping/provider.go

```diff
@@ -60,7 +62,10 @@ func (provider *provider) Drop(ledgerID string) error {
 	if err := provider.dbProvider.Drop(dbName(ledgerID, PvtdataExpiry)); err != nil {
 		return err
 	}
-	return provider.dbProvider.Drop(dbName(ledgerID, MetadataPresenceIndicator))
+	if err := provider.dbProvider.Drop(dbName(ledgerID, MetadataPresenceIndicator)); err != nil {
+		return err
+	}
+	return provider.dbProvider.Drop(dbName(ledgerID, SnapshotInterest))
```

Comment:
perhaps this can be replaced by `return provider.dbProvider.Drop(ledgerID)`, so this does not need a change when we add another category.

Reply:
Talked to Manish offline and it was agreed to leave `Drop` as is.